app/globals.css

```
@@ -45,159 +45,190 @@ body {
+    135deg,
+    #0a0a0f 0%,
+    #1a1a2e 25%,
+    #16213e 50%,
+    #0f3460 75%,
+    #0a0a0f 100%
+  );
+  background-size: 400% 400%;
+  animation: gradientShift 15s ease infinite;
+}
+
+@keyframes gradientShift {
```

Stylelint (17.11.1)

[error] Expected keyframe name "gradientShift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)